src/Monomer/Core/Combinators.hs

```diff
@@ -350,7 +350,7 @@ class CmbThumbFactor t where
 class CmbThumbRadius t where
   thumbRadius :: Double -> t
 
--- | Whether the thumb is visible, for example in a scroll.
+-- | Whether the thumb is visible, for example in a slider.
```

Comment:
This one was correct! 😄

Reply:
Well, only `SliderCfg` is an instance of the `CmbThumbVisible` class.